source/Ship.cpp

```diff
@@ -125,6 +125,8 @@ void Ship::Load(const DataNode &node)
 				armament.AddGunPort(hardpoint, outfit);
 			else
 				armament.AddTurret(hardpoint, outfit);
+			if (armament.Get().size() > 32)
+				child.PrintTrace("Warning: Ships with more than 32 hardpoints do not behave as one would expect:");
```

Comment:
I changed both things as requested.

I personally liked it a bit to see that wall of lines. Giving me a better feeling of intensity.
But I understand your argument and accept it.